hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/io/storage/row/parquet/ParquetRowDataWriter.java

```diff
@@ -283,34 +291,66 @@ public void write(ArrayData array, int ordinal) {
     }
   }
 
-  /**
-   * Timestamp of INT96 bytes, julianDay(4) + nanosOfDay(8). See
-   * https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#timestamp
```

Comment:
Can we fix the doc instead of removing it.

Reply:
Done